src/TodoListItem.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const TodoListItem = (props) => (
+    <li key={props.item.id}>{props.item.title}</li>
```

Comment:
you actually don't need the key here since the parent component (the `TodoListItem`) has a key